### ISSUE TITLE
Organize list of slices in a project by which are your slices

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#710]
Project leads in particular see lots of slices in the project.

Put slices that you are the owner for first? Or in a separate table?

Maybe more generally group slices by your role, with slices you are lead for first?

 _Imported from trac ticket [#710](http://trac.gpolab.bbn.com/proto-ch/ticket/710),  created by ahelsing on 07-11-2013 at 15:53, last modified: 12-16-2013 at 14:35_

**Comment by ahelsing** [comment]
Talk to experimenter support about whether this is a good idea, and what is the best implementation

_Trac comment by phelinek on 12-11-2013 at 11:13_

**Comment by seledwards** [comment]
I love this idea.  I've been having trouble with this lately and I think it is likely to be a problem for any project lead.

I personally like: "Maybe more generally group slices by your role, with slices you are lead for first?"


_Trac comment by sedwards (github user: seledwards) on 12-11-2013 at 11:34_

**Comment by ahelsing** [comment]
I talked to Sarah and she thought best would be to organize into four tables by slice role.  However, we only know the lead and determining other roles would require an additional query for each slice, slowing things down if you are a member on a lot of slices.  So we decided to have two tables, one for slices on which you are lead and another for slices on which you are not lead.

The tables are under the heading "My Slices", and originally I labeled them "My Slices" and "Other Slices" but I didn't like that because both tables aren't always there.  So when you own all the slices you had: Heading- "My Slices", Subheading -"My Slices".  ANd if you didn't own any slices you had: Heading - My Slices, Subheading "Other Slices".  Neither case looked good.  Instead I made the subheadings: "Slices on which I am lead" and "Slices on which I am not lead".  Clear but wordy.  I can change them if people have better suggestions.

Then I thought it was confusing the the Slice Lead column was called Slice Owner.  I thought we should use the terms consistently and either say Lead or Owner, but not both in the same table.  So I changed the column to Slice Lead.  If that's a problem I can change it back.

_Trac comment by phelinek on 12-16-2013 at 14:35_
